mlflow/store/tracking/file_store.py

```diff
@@ -39,6 +39,7 @@
     _validate_batch_log_data,
     _validate_list_experiments_max_results,
     _validate_param_keys_unique,
+    _validate_experiment_name as validate_experiment_name,
```

Comment:
Do we need the as part? I think `_validate_experiment_name` and `FileStore._validate_experiment_name` don't conflict.

Reply:
that was my intention. Do you think I should refactor the internal module name instead?

Reply:
Offline discussion:
- remove `as validate_experiment_name `
- rename `FileStore._validate_experiment_name` to `FileStore._validate_experiment_does_not_exist` and apply it and `_validate_experiment_name` separately.